site/content/en/docs/manual/advanced/automatic-annotation.md

```diff
@@ -2,54 +2,106 @@
+| YOLO v5                   | YOLO v5 is a family of object detection architectures and models based on the Pytorch framework. <br><br> For more information, see: <li>[GitHub: YOLO v5](https://github.com/ultralytics/yolov5) <li>[Site: YOLO v5](https://docs.ultralytics.com/#yolov5)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 
-   ![](/images/image121_detrac.jpg)
+<!--lint enable maximum-line-length-->
 
-1. The end result of an automatic annotation is an annotation with separate rectangles (or other shapes)
+## Adding models from Hugging Face and Roboflow
 
-   ![](/images/gif014_detrac.gif)
+In case you did not find the model you need, you can add model
+of your choice from [Hugging Face](https://huggingface.co/)
+or [Roboflow](https://roboflow.com/).
 
-1. You can combine separate bounding boxes into tracks using the `Person reidentification ` model.
-   To do this, click on the automatic annotation item in the action menu again and select the model
-   of the `ReID` type (in this case the `Person reidentification` model).
-   You can set the following parameters:
+<!--lint disable maximum-line-length-->
 
-   - Model `Threshold` is a maximum cosine distance between objects’ embeddings.
-   - `Maximum distance` defines a maximum radius that an object can diverge between adjacent frames.
+For more information,
+see [Streamline annotation by integrating Hugging Face and Roboflow models](https://www.cvat.ai/post/integrating-hugging-face-and-roboflow-models).
```

Comment:
This doesn't seem right to me. This page is part of the reference manual, it should be the _source_ of all information. It's blog articles that should refer _here_ for more information.

Reply:
We do not have integration documentation for now. And it must be done separately as a separate task. For now, the link is to the blog article because we do not have anything better

Reply:
Okay.